src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -415,7 +415,7 @@ public void testGetCaseByCaseId_caseSPGhandDeliveryFalse() {
   }
 
   @Test
-  public void shouldGetSecureEstablishmentByCaseId() {
+  public void shouldGetSecureEstablishmentByCaseId() throws CTPException {
```

Comment:
I think we need at least one extra unit test that gets a case by ID, and returns it from the cache.

Reply:
Test case added